pymbar/testsystems/exponential_distributions.py

```diff
@@ -23,18 +23,19 @@ class ExponentialTestCase(object):
 
     Generate energy samples with default parameters in one line.
 
-    >>> [x_kn, u_kln, N_k] = ExponentialTestCase().sample()
+    >>> x_kn, u_kln, N_k = ExponentialTestCase().sample()
 
     Generate energy samples with specified parameters.
 
     >>> testcase = ExponentialTestCase(rates=[1., 2., 3., 4., 5.])
-    >>> [x_kn, u_kln, N_k] = testcase.sample(N_k=[10, 20, 30, 40, 50])
+    >>> x_kn, u_kln, N_k = testcase.sample(N_k=[10, 20, 30, 40, 50])
 
     Test sampling in different output modes.
 
-    >>> [x_kn, u_kln, N_k] = testcase.sample(N_k=[10, 20, 30, 40, 50], mode='u_kln')
-    >>> [x_n, u_kn, N_k, s_n] = testcase.sample(N_k=[10, 20, 30, 40, 50], mode='u_kn')
-    >>> [w_F, w_R, N_k] = testcase.sample(N_k=[40, 50], mode='wFwR')
+    >>> x_kn, u_kln, N_k = testcase.sample(N_k=[10, 20, 30, 40, 50], mode='u_kln')
+    >>> x_n, u_kn, N_k, s_n = testcase.sample(N_k=[10, 20, 30, 40, 50], mode='u_kn')
+    >>> testcase2 = ExponentialTestCase(rates=[4., 5.])
+    >>> w_F, w_R, N_k = testcase2.sample(N_k=[40, 50], mode='wFwR')
```

Comment:
Does this fix make sense? We need a 2nd test case so `len(rates) == len(N_k)`

Reply:
OK, added something to test both cases.  Reused 'testcase' since it's just a test variable.